rules/empty-brace-spaces.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const getDocumentationUrl = require('./utils/get-documentation-url');
+
+const MESSAGE_ID = 'empty-brace-spaces';
+const messages = {
+	[MESSAGE_ID]: 'Do not add spaces between braces.'
+};
+
+const selector = `:matches(${
```

Comment:
Isn't it easier to just do this?

```js
const selector = ":matches(BlockStatement[body.length=0], ClassBody[body.length=0], ObjectExpression[properties.length=0])"
```

Reply:
Yes, same. But I prefer current one, it's more readable add diff friendly, if we want add more.

```diff
const selector = `:matches(${
	[
		'BlockStatement[body.length=0]',
		'ClassBody[body.length=0]',
+		'ImportNamespaceSpecifier[assertions.length=0]',
		'ObjectExpression[properties.length=0]'
	].join(', ')
})`;
```

VS

```diff
- const selector = ":matches(BlockStatement[body.length=0], ClassBody[body.length=0], ObjectExpression[properties.length=0])"
+ const selector = ":matches(BlockStatement[body.length=0], ClassBody[body.length=0], ImportNamespaceSpecifier[assertions.length=0], ObjectExpression[properties.length=0])"